libraries/ms-common/inc/soft_timer.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <stdint.h>
+
+#include "status.h"
+
+#define SOFT_TIMER_NUM 10
+
+typedef void (*soft_timer_callback)(uint8_t timer_id, void* context);
```

Comment:
I don't have a strong preference in this case. To me PascalCase mean struct/class/enum but I'm okay with that also applying to callbacks.